perllib/Open311/GetServiceRequests.pm

```diff
@@ -157,6 +157,7 @@ sub create_problems {
         my $state = $open311->map_state($request->{status});
 
         my $non_public = $request->{non_public} ? 1 : 0;
+        $non_public ||= $contacts[0] ? $contacts[0]->non_public : 0;
```

Comment:
This is fine, just noting that it means if a Open311 endpoint explicitly *does* say it is public, this will still override it. Presumably that case would never happen, so not worth checking for it being defined at all?

Reply:
You're right, it's very unlikely to happen! I think it's safer to be able to say 'all fetched reports in a `non_public` category will be marked as `non_public`'.